index.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+class Application
+{
+
+    function run()
+    {
+        $times = 0;
+        $historyRepository = new HistoryRepository();
+        $date = date('d-m-Y');
+        info_box('', 'Welcome to the calculator app!', '', 'Print "help" to learn more about the app.', '');
+
+        while ($times < 1) {
+
+            $command = choose('Enter command: ', AVAILABLE_COMMANDS);
+
+            if (in_array($command, SYSTEM_COMMANDS)) {
+                $this->execute_system_command($command, $historyRepository);
+
+                continue;
+            }
+
+            $argument1 = $this->read_operand('Enter fist number: ', $command);
+
+            if ($command != '^' && $command != 'sr' ) {
+                $argument2 = $this->read_operand('Enter second number: ', $command, true);
+            } else {
+                $argument2 = $this->read_operand('Enter exponent: ', $command);
+            }
+
+            $result = $this->calculate($argument1, $command, $argument2);
+
+            info('Result: ' . $result);
+            info('=====================');
+
+            $historyRepository->create($date, $argument1, $argument2, $command, $result);
+        }
+    }
+    function calculate($argument1, $command, $argument2)
+    {
+        switch ($command) {
+            case '+':
+                return $argument1 + $argument2;
+            case '-':
+                return $argument1 - $argument2;
+            case '*':
+                return $argument1 * $argument2;
+            case '/':
+                return $argument1 / $argument2;
+            case '^':
+                return pow($argument1, $argument2);
+            case 'sr':
+                return pow($argument1, (1 / $argument2));
+            default:
+                return $command;
+        }
+    }
+
+    function read_operand($message, $command, $isSecondOperand = false)
+    {
+        do {
+            $argument = readline($message);
+            $int1 = str_to_number($argument);
+            $isDataValid = ($argument == $int1);
+
+            if (!$isDataValid) {
+                info('Cant write letters.');
+
+                continue;
+            }
+
+            $isDataValid = ($argument != null);
+
+            if (!$isDataValid) {
+                info('Cant write space.');
+
+                continue;
+            }
+
+            if ($isSecondOperand) {
+                $isDataValid = ($command != '/' || $argument != 0);
+
+                if (!$isDataValid) {
+                    info('Cant separate on zero.');
+                }
+            }
+
+        } while (!$isDataValid);
+
+        return $argument;
+    }
+
+    function execute_system_command($command, $historyRepository)
+    {
+        switch($command) {
+            case(QUIT):
+                $this->finish_app();
+
+                break;
+
```

Comment:
please also apply this fix to other cases

```suggestion
                return $this->finishApp();
```